object_script.SolarCarTelemetry.Debug

```diff
@@ -0,0 +1,122 @@
+INPUT(
```

Comment:
These build files got into the repository. Remove these from git and add to ignore.

These are now suddenly getting created because the linking command line is now too long to actually do it all on a single command line... so to get around this qmake makes these scripts. 

I finished my computer graphics assignment using the subdirs option within qmake which would fix this problem. So that is mostly working now and you can expect a pull request within the next week. The only thing I have left to figure out is sometimes the target does not get relinked even if the library changes...